browser/ui/webui/settings/default_brave_shields_handler.cc

```diff
@@ -9,13 +9,34 @@
 #include "base/bind.h"
 #include "base/values.h"
 #include "brave/common/pref_names.h"
+#include "brave/components/brave_shields/browser/brave_shields_util.h"
 #include "brave/components/brave_shields/common/brave_shield_constants.h"
 #include "chrome/browser/content_settings/host_content_settings_map_factory.h"
 #include "chrome/browser/profiles/profile.h"
 #include "components/content_settings/core/browser/host_content_settings_map.h"
 #include "components/prefs/pref_service.h"
 #include "content/public/browser/web_ui.h"
 
+using brave_shields::ShieldState;
+
+namespace {
+
+ShieldState ShieldStateFromString(base::StringPiece value) {
+  if (value == "allow") {
+    return ShieldState::kAllow;
+  }
+  if (value == "3p") {
```

Comment:
This is defined as `"block3rd"` in `components/brave_shields/common/brave_shield_switches.h`. I think they should be the same string, no?

Reply:
I wonder if more simple to just document the same values and force command line usage by those values? Might be less confusing and avoid the hardcoded strings.

Reply:
`3p` sounds quite confusing for me: is it allow 3p or block 3p? So I'm not up to using it as a command line switch. I've already removed a bunch of hardcoded usages of this in the file, so isolating this `3p` in a function at least looks better